keps/sig-api-machinery/1040-priority-and-fairness/README.md

```diff
@@ -3013,10 +3013,10 @@ GA:
 
 - [x] API annotations properly support strategic merge patch.
 - [x] Design and implement borrowing between priority levels.
-- [ ] Satisfaction with LIST and WATCH support.
-- [ ] Implement borrowing by exempt priority levels.
-- [ ] APF allows us to disable client-side rate limiting without causing the apiservers to wedge/crash.  Note that there is another level of concern that APF does not attempt to address, which is mismatch between the throughput that various controllers can sustain.
-- [ ] Satisfaction that the interface is sufficient to support tuning of capacity and resource costs.
+- [x] Satisfaction with LIST and WATCH support.
+- [x] Implement borrowing by exempt priority levels.
+- [x] APF allows us to disable client-side rate limiting without causing the apiservers to wedge/crash.  Note that this falls a bit short of saying that we would actually want to disable client-side rate limiting.  While experiments show that APF can prevent server crashes, they do not show that the SLOs are met in all scenarios.  If the control plane is under-provisioned, something has to give.
```

Comment:
nit: "... something has to give" doesn't parse to me.

Reply:
Oh, perhaps that is local idiom.  I will reword.